CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 All notable changes to this project will be documented in this file.
 
+## \[6.4.5\]
```

Comment:
I think this should also include https://github.com/GoogleCloudPlatform/slurm-gcp/pull/111. Have you reviewed history to make sure this is inclusive?

Reply:
I missed that due to reviewing my local working copy. I've added it. It's important!